where-to-travel/src/main/webapp/script.js

```diff
@@ -1,141 +1,302 @@
-//This is map stylings for the GMap api
-const mapStyles = [
+// This is map stylings for the GMap api
+const MAP_STYLES = [
   {
-    "featureType": "landscape",
-    "stylers": [
+    featureType: 'landscape',
+    stylers: [
       {
-        "visibility": "off"
+        visibility: 'off'
       }
     ]
   },
   {
-    "featureType": "poi",
-    "elementType": "labels.icon",
-    "stylers": [
+    featureType: 'poi',
+    elementType: 'labels.icon',
+    stylers: [
       {
-        "visibility": "off"
+        visibility: 'off'
       }
     ]
   },
   {
-    "featureType": "transit",
-    "stylers": [
+    featureType: 'transit',
+    stylers: [
       {
-        "visibility": "off"
+        visibility: 'off'
       }
     ]
   }
 ];
-//End map stylings
 
-const submitId = "submit-id";
-const hoursId = "hrs";
-const minutesId = "mnts";
+// Thresholds for termination of search algorithm
+const PLACES_THRESHOLD = 30;
+const ATTEMPTS_THRESHOLD = 10;
+const DIRECTION_THRESHOLD = 5;
+
+// Document ids for user input elements
+const SUBMIT_ID = 'submit';
+const HOURS_ID = 'hrs';
+const MINIUTES_ID = 'mnts';
+const SCROLL_ID = 'scroller';
+const DASH_ID = 'dashboard';
+const PIN_PATH = 'icons/pin.svg';
+const SELECTED_PIN_PATH = 'icons/selectedPin.svg';
+const HOME_PIN_PATH = 'icons/home.svg';
 
 let map;
+let user = false;
 let home = null;
 
+let focussedCard;
```

Comment:
nit spelling:
let focusedCard;
let focusedPin;

Reply:
fixed in 405fa36